### PR TITLE
[Magic Firewall] Added Plans page

### PR DESCRIPTION
**Pull request by dcpena**
Added a Plans page to describe the difference in standard vs. advanced features.

Addresses [PCX-3440](https://jira.cfops.it/browse/PCX-3440).

### CODE REVIEW

File: content/magic-firewall/plans.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Plans
+pcx-content-type: concept
+weight: 2
+---
+
+# Plans
+
+If you are a [Magic Transit](/magic-transit/) or [Magic WAN](/magic-wan/) user, you are automatically provided with a standard list of Magic Firewall features. For additional features available for purchase, see the list of advanced features below.
```
Comment (kodster28) [inline content/magic-firewall/plans.md]:
```suggestion
If you are a [Magic Transit](/magic-transit/) or [Magic WAN](/magic-wan/) user, you are automatically provided with a standard list of Magic Firewall features. For additional features available for purchase, refer to the list of advanced features below.
```

File: content/magic-firewall/plans.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+### Standard features
+
+- Filtering rules based on protocol, port, IP addresses, packet length and bit field match
```
Comment (kodster28) [inline content/magic-firewall/plans.md]:
```suggestion
- Filtering rules based on protocol, port, IP addresses, packet length, and bit field match
```

File: content/magic-firewall/plans.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- Fast propagation of rule changes in <500ms
+- Single dashboard to manage firewall and network configuration
+- Programmable API for automated deployment and management — compatible with infrastructure-as-code platforms like Terraform
```
Comment (kodster28) [inline content/magic-firewall/plans.md]:
```suggestion
- Programmable API for automated deployment and management — compatible with infrastructure-as-code platforms like [Terraform](/terraform/)
```

File: content/magic-firewall/plans.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+- Traffic analytics per rule in the dashboard and using the GraphQL API
```
Comment (kodster28) [inline content/magic-firewall/plans.md]:
```suggestion
- Traffic analytics per rule in the dashboard and using the [GraphQL API](/analytics/graphql-api/)
```

File: content/magic-firewall/plans.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+- Integration with Magic WAN network-as-a-service
```
Comment (kodster28) [inline content/magic-firewall/plans.md]:
```suggestion
- Integration with [Magic WAN network-as-a-service](/magic-wan/)
```

File: content/magic-firewall/plans.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+- Included DDoS protection with Magic Transit
```
Comment (kodster28) [inline content/magic-firewall/plans.md]:
```suggestion
- Included DDoS protection with [Magic Transit](/magic-transit/)
```

File: content/magic-firewall/plans.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+### Advanced features
+
+- Customizable IP lists
+- Managed threat intelligence IP lists (Malware, Botnet, Anonymizer)
+- Geoblocking based on user location by country
+- Packet captures on demand for network troubleshooting
+- Protocol validation rules to inspect traffic validity and enforce a positive security model
+- Optional upgrade to full stateful Secure Web Gateway using Cloudflare Zero Trust for outbound Internet traffic
```
Comment (kodster28) [inline content/magic-firewall/plans.md]:
```suggestion
- Optional upgrade to full stateful Secure Web Gateway using [Cloudflare Zero Trust](/cloudflare-one/) for outbound Internet traffic
```

**Review by kodster28 — APPROVED** [review]

File: content/magic-firewall/plans.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+title: Plans
+pcx-content-type: concept
+weight: 2
+---
+
+# Plans
+
+If you are a [Magic Transit](/magic-transit/) or [Magic WAN](/magic-wan/) user, you are automatically provided with a standard list of Magic Firewall features. For additional features available for purchase, see the list of advanced features below.
+
+### Standard features
```
Comment (pedrosousa) [inline content/magic-firewall/plans.md]:
```suggestion
## Standard features
```

File: content/magic-firewall/plans.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+- Filtering rules based on protocol, port, IP addresses, packet length and bit field match
+- Fast propagation of rule changes in <500ms
+- Single dashboard to manage firewall and network configuration
+- Programmable API for automated deployment and management — compatible with infrastructure-as-code platforms like Terraform
+- Traffic analytics per rule in the dashboard and using the GraphQL API
+- Integration with Magic WAN network-as-a-service
+- Included DDoS protection with Magic Transit
+
+### Advanced features
```
Comment (pedrosousa) [inline content/magic-firewall/plans.md]:
```suggestion
## Advanced features
```

File: content/magic-firewall/plans.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+- Filtering rules based on protocol, port, IP addresses, packet length, and bit field match
+- Fast propagation of rule changes in <500ms
+- Single dashboard to manage firewall and network configuration
+- Programmable API for automated deployment and management — compatible with infrastructure-as-code platforms like [Terraform](/terraform/)
+- Traffic analytics per rule in the dashboard and using the [GraphQL API](/analytics/graphql-api/)
+- Integration with [Magic WAN network-as-a-service](/magic-wan/)
+- Included DDoS protection with [Magic Transit](/magic-transit/)
+
+## Advanced features
+
```
Comment (arges) [inline content/magic-firewall/plans.md:22]: I would suggest mentioning that standard features are also included.

**Review by arges — CHANGES_REQUESTED** [review]

File: content/magic-firewall/plans.md (hunk 10 above)
Comment (dcpena) [reply content/magic-firewall/plans.md:22]: Good callout. Added that info

**Comment by dcpena** [comment]
Sent Annika the preview link and she approved. Mind taking one more pass, @arges?

**Review by arges — APPROVED** [review]

**Pull request merged**